packages/@orchidui-vue/src/DataDisplay/Table/OcTableCell.vue

```diff
@@ -63,20 +63,6 @@ const variantClass = computed(() => ({
   [Variants.DATETIME]: "px-4",
   [Variants.EMPTY]: "px-4 min-w-[48px]",
 }));
-const isCopied = ref(false);
```

Comment:
@mykhailo-diakovych are you sure removing isCopy, will not be a problem?

Reply:
Yes, cause we already had this logic in CopyTooltip which is using in TableCell